docker-compose.yml

```diff
@@ -163,9 +163,9 @@ services:
       context: .
       dockerfile: Dockerfile.database
     expose:
-      - "3306"
+      - "3307"
```

Comment:
Could you revert the changes on this file?

Reply:
Sure z let me do it!